docs/python.org/code-of-conduct/Transparency-Report-Guidelines.md

```diff
@@ -0,0 +1,45 @@
+
+In this document we're discussing the **public-facing** summary transparency report. For operational purposes, an event should also keep private records of all reports and issues at the event.
+
+
+## When to release
+
+A timely report helps everybody:
+
+- Reporters know that their reports were taken seriously;
+- Attendees can see that CoC reports are an integral part and so feel confident to attend in the future and report issues they themselves witness;
+- Organizers demonstrate strong commitment to follow their stated goals and procedures.
+
+Since it's important to be able to recall details of covered issues accurately, a timely report will be of higher quality.
+
+Releasing an initial transparency report should be part of the closing ceremony. Following up with affected parties might take time after the event. Therefore, a separate complete report can be provided in writing at a later time. It wouldn't take more than a month after the event to close the loop on the final report.
+
+
+## How to publish
+
+An initial **in-person report** during the closing ceremony is highly recommended. Any information available by that time, including statistics on the number of reports, will be very valuable to attendees at the event. Cases that need following up shouldn't block the initial report from being presented.
+
+After the event, any **official communication channel** used with the attendees is a good option for letting people know a transparency report is available. Examples include:
+
+- Official website for the event;
+- Official blog for the event and/or organization behind it;
+- Newsletter or mailing list(s) in use around the event.
+
+
+## What to include
+
+Starting with some general information about the event puts all data in context. Include **the size of the event**, with separate numbers for in-person and online attendees, if applicable. Briefly describe the **reporting process**; who the people enforcing the CoC and responding to reports were; how an attendee could report a CoC issue; and where this information was available for attendees to find.
+
+The main part of the report will be the **statistics of reports** received (including past 3~5 years historical statistics can be helpful for comparison) and the **report status / progress for each report**.
+
+If there was a post-conference survey asking attendees about their confidence in reporting issues, the results of that survey can be included in the following year's transparency report.
+
+
+## What **NOT** to include
+
+Make sure the transparency report does not include any sensitive information such as the names or any identifiable information of the reporters and the reported people. On the off-chance that local police were called, you should say they were called and not make further comments on the details as it could adversely affect official action. 
```

Comment:
```suggestion
Make sure the transparency report does not include any sensitive information such as the names or any identifiable information of the reporters and the reported people. In the event that law enforcement was called, you should say they were called and not make further comments on the details as it could adversely affect official action. 
```

Remove the idiom for clarity and also generalize to any law enforcement because sometimes you might have federal crimes or you might have your event in a location that only has national police.